src/main/java/xyz/luan/facade/HttpFacade.java

```diff
@@ -40,10 +45,32 @@
 
     private List<InputStream> customKeystoreCertificates = new ArrayList<>();
 
+    private void allowNewerMethods(String... methods) {
```

Comment:
My understanding is that this has to be run only once ever, right? by calling on the constructor you will make it called every new instance. In order to do something like this I see two options:

1. if we want the hack to run every time on startup, no option against it, we can do a static initializer:
```
public class HttpFacade {

static {
    your code
}
```

2. if we want to run this on demand this can be a static method on HttpFacade each application has to decide to call on its startup.

I'm actually ok with option 1, because this is a very important fix, and the fact that is a hack is a severe limitation of the platform. but then I would change one thing: on the try catch, just log the error. normally I don't like to swallow exceptions but in this case, if this code doesn't work, the user should be able to use the library without an uncatchable application catch. it will still work on all other verbs.

what do you think?

Reply:
I agree with you.
I implemented the code in the initializer static, so it will be loaded only once.
Also, I tested the exception part and other verbs work normally.